recipes/flasgger/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flasgger" %}
+{% set version = "0.8.0" %}
+{% set md5 = "185f95e37f3d8cfc90f4eb80be0f8f72" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Is this package pure-python? If so, please add:

```yml
build:
  noarch: python
```

So this becomes a pure python package.

Reply:
Done